### DISCUSSION TITLE
[Idea] Add container UI element to field layout designer

### DISCUSSION THREAD

**Discussion by wsydney76** [craftcms/cms#8900] [Ideas]
### Description

The new field layout designer is highly appreciated, and one thing came up to make it even better:

Add a field container containing several fields, that can be set to a custom width as a whole. (plus heading, show border?)

Use case 1:

Align a couple of settings beneath an input field:

![image](https://user-images.githubusercontent.com/60010597/90315383-659b2980-df1b-11ea-8482-98b3e51cf8e0.png)

Use case 2:

Group inputs together in order to preserve a meaningful order if there is no space for multiple columns (preview, HUD)

E.g two columns block

![image](https://user-images.githubusercontent.com/60010597/90315687-a2682000-df1d-11ea-9987-bf10e42e9efe.png)

Order in preview would be now: Image left, Image right, Heading left, Heading right, Text left, Text right

Better with containers: Image left, Heading left, Text left    Image right Heading right Text right

**Comment by piotrpog** [comment]
That's how I sometimes use matrix field - I limit it to just 1 block and put some fields inside to group them.

**Comment by brandonkelly** [comment]
Wondered how long it would take before this came up :)

This would also provide a nice solution for breaking up rows of fields (#6426).
